nuxt.config.ts

```diff
@@ -113,11 +121,11 @@ export default defineNuxtConfig({
   runtimeConfig: {
     public: {
       recaptcha: {
-        siteKey: process.env.RECAPTCHA_SITE_KEY,
+        siteKey: '',
       },
-      baseUrl: process.env.BASE_URL || '',
-      maintenance: process.env.MAINTENANCE || 'false',
-      testing: process.env.TESTING,
+      baseUrl: '',
+      maintenance: 'false',
+      testing: 'false',
```

Comment:
we can't use boolean instead here?

Reply:
Okay I see, it only accept string

Reply:
> Okay I see, it only accept string

Yes, I tried to put a boolean there in the beginning :'(